src/sdk/service/upload-service/upload-service.ts

```diff
@@ -188,6 +188,7 @@ export abstract class UploadService {
           });
         });
       }
+      /* tslint:disable-next-line */
```

Comment:
what did you need to disable on this line @ariquinones ?

Reply:
it was a lint error as well saying you don't use that variable

Reply:
const client = this.getHTTPClient();

Reply:
ok, we just need to remove the line vs. ignore it then. (and thanks for fixing my lint errors)

Reply:
Okay updated